### PR TITLE
Enable map reports

### PR DESCRIPTION
**Pull request by kaapstorm**
This PR changes default page size to 100 for map reports, adds a notice if data extends beyond a single page, and drops the toggle.

The pagination notice looks like so:

pagination_notice

More context at [Trello](https://trello.com/c/LutdwNCP/31-handle-paging-in-maps-and-release).

One digression from requirements is that I placed the notice where other messages are shown, but I'm happy to move it to between the map and the table if people feel that's a better place.

@NoahCarnahan cc @sheelio @czue @proteusvacuum

### CODE REVIEW

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 1]
```diff
@@ -58,6 +58,15 @@
             }
         };
 
+        var paginationNotice = function (data) {
+            if (data.aaData.length < data.iTotalRecords) {
+                $('#info-message').html("Showing the current page of data. Switch pages to see more data.");
```
Comment (proteusvacuum) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: should this be translated? I'd maybe also change `the current page` to read `a single page`. 
(Aside: this does seem like a bit of a roundabout way of solving the problem that people aren't seeing the pagination controls (if that is the issue), might be worth putting a pager at the top as well?)

Comment (kaapstorm) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: :+1: Added template tag for translation. I'll leave copywriting choices to @sheelio.

Good point about the UI. I can see how paging might be less obvious with maps than with charts, but maybe paging controls should be at the top and the bottom for all reports.

**Comment by sheelio** [comment]
@kaapstorm I would move it below the map (for visibility).  

Paging controls below/above list would be nice, but that seems like a big change to HQ.

**Comment by czue** [comment]
lgtm. is there any additional doc work or anything we need to do before we launch this?

**Comment by sheelio** [comment]
I think good to release - I'll update the documentation.

**Pull request merged**